packages/astro/src/default/utils/content.ts

```diff
@@ -239,11 +249,11 @@ export async function getTutorial(): Promise<Tutorial> {
     const chapterMetadata = _tutorial.parts[lesson.part.id].chapters[lesson.chapter.id].data;
 
     lesson.data = {
-      ...pick(
+      ...lesson.data,
+      ...squash(
         [lesson.data, chapterMetadata, partMetadata, tutorialMetaData],
-        ['mainCommand', 'prepareCommands', 'previews', 'autoReload', 'template', 'terminal', 'editor', 'focus'],
+        ['mainCommand', 'prepareCommands', 'previews', 'autoReload', 'template', 'terminal', 'editor', 'focus', 'i18n'],
```

Comment:
Question: does this achieve the inheritance and overrides of metadata?

Reply:
It does now with the rework of `pick` -> `squash` :partying_face:

Reply:
Awesome, perfect timing for my PR! I'll take a look at this part closer. 😄 

https://github.com/AriPerkkio/tutorialkit/commit/68795c1c5029e339fd3e45f180f59924f734fd94#diff-429634fe12d5be6dc111f0d4ae106edc755b2dbae9bb521b4c30fe51447ac57bR84-R93